jiant/utils/utils.py

```diff
@@ -33,6 +33,20 @@
 _MOSES_DETOKENIZER = MosesDetokenizer()
 
 
+def get_output_attribute(out, attribute_name):
+    if torch.cuda.device_count() > 1:
+        return out[attribute_name].sum()
```

Comment:
Is it always the case that you can call sum (or want to call it) on the output? For loss that makes sense, but not sure it makes sense in general.

Reply:
Yes, that's true. For our current use cases (loss, n_exes) it's true, but for other people who use this they might have to tweak this function to suit their needs.

Reply:
This seems okay to me as is, but make sure to add a docstring that makes it clear that you use sum, and that this won't be appropriate for non-summed attributes like accuracy.